src/gc-stacks.c

```diff
@@ -73,6 +73,9 @@ static void free_stack(void *stkbuf, size_t bufsz)
 }
 #endif
 
+JL_DLLEXPORT uint32_t jl_get_num_stack_mappings() {
+    return jl_atomic_load_relaxed(&num_stack_mappings);
+}
```

Comment:
Should this also be exposed through a julia function in Base? Or is it okay to just expose the ccall like this?

@c42f: Are you the expert now on what "public" / "exported" means?

Reply:
I think keeping it a ccall is fine